bazel/api_build_system.bzl

```diff
@@ -77,7 +77,7 @@ def api_go_grpc_library(name, proto, deps = []):
 # gRPC stub generation.
 # TODO(htuch): Automatically generate go_proto_library and go_grpc_library
 # from api_proto_library.
-def api_proto_library(name, srcs = [], deps = [], has_services = 0, require_py = 1):
+def api_proto_library(name, visibility, srcs = [], deps = [], has_services = 0, require_py = 1):
```

Comment:
@PiotrSikora as long as this works on the main Envoy repo I think it's OK. We clobber this during import, so it's not an issue inside Google.